pkg/ipamd/ipamd.go

```diff
@@ -547,23 +547,25 @@ func (c *IPAMContext) nodeInit() error {
 		}, 30*time.Second)
 	}
 
-	node, err := k8sapi.GetNode(ctx, c.k8sClient)
-	if err != nil {
-		log.Errorf("Failed to get node", err)
-		podENIErrInc("nodeInit")
-		return err
-	}
-
-	eniConfigName, err := eniconfig.GetNodeSpecificENIConfigName(node)
-	if err == nil && c.useCustomNetworking && eniConfigName != "default" {
-		// Add the feature name to CNINode of this node
-		err := c.AddFeatureToCNINode(ctx, rcv1alpha1.CustomNetworking, eniConfigName)
+	if c.useCustomNetworking {
```

Comment:
nit: Let's add a comment here that when Custom Networking is enabled and a valid ENIConfig is found, IPAMD patches the `CNINode` resource for this instance. This operation is safe as enabling/disabling Custom Networking requires terminating the previous instance.